modules/tracktion_engine/tracktion_engine.h

```diff
@@ -630,6 +630,9 @@ namespace tracktion { inline namespace engine
 #include "plugins/effects/tracktion_Phaser.h"
 #include "plugins/effects/tracktion_PitchShift.h"
 #include "plugins/effects/tracktion_Reverb.h"
+//
+#include "../Source/Plugin/bc_EffectsModule.h" // Must be included before SamplerPlugin as it composes EffectsModule.
```

Comment:
This has to go. Why do we still have DLL dependencies in tracktion?

Reply:
The issue for this particular instance will go away when we move all the modified SamplerPlugin code into a new class, but the include is there because otherwise Tracktion Engine won't compile.